conf/application.conf

```diff
@@ -29,6 +29,7 @@ location.canonical.list{
 # ~~~~
 # Additional play modules can be added here
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
```

Comment:
Do we need to remove the old v1 module of HttpClient or is that still required alongside v2?

Reply:
I don't think it should be needed, but I have just noticed upon removing it there's a few I have not converted to V2, good spot!